### PR TITLE
ampproject: initial integration

### PR DESCRIPTION
**Pull request by DavidKorczynski**

### CODE REVIEW

**Review by jonathanmetzman — APPROVED** [review]
lgtm

File: projects/ampproject/build.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+git apply  --ignore-space-change --ignore-whitespace $SRC/fuzz_patch.diff
```
[the remark this answers is not in the document]
Comment (DavidKorczynski) [reply projects/ampproject/build.sh]: Sounds good, done!

File: projects/ampproject/build.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -eu
```
[the remark this answers is not in the document]
Comment (DavidKorczynski) [reply projects/ampproject/build.sh:1]: Yes, thanks! Done

File: projects/ampproject/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+RUN apt-get update && apt-get install -y make autoconf automake libtool libomp-dev libgomp1 nodejs
+RUN git clone --depth 1 https://github.com/ampproject/amphtml amphtml
+#COPY amphtml amphtml
```
[the remark this answers is not in the document]
Comment (DavidKorczynski) [reply projects/ampproject/Dockerfile]: Yes, thanks! Done

**Pull request merged**